sound/soc/sof/ipc.c

```diff
@@ -172,6 +172,9 @@ struct snd_sof_ipc *snd_sof_ipc_init(struct snd_sof_dev *sdev)
 		return NULL;
 	}
 
+	if (!ops)
+		return NULL;
+
```

Comment:
You would also need to
```c
const struct sof_ipc_ops *ops = NULL;
```
but, this is a false alarm. `ops` will be set _or_ we will return from the default case, I don't see other option.

Reply:
ok ill drop this patch then.

Reply:
I run ccpcheck locally and it found no issue with ipc.c:
```
Checking sound/soc/sof/ipc.c ...
Checking sound/soc/sof/ipc.c: CONFIG_SND_SOC_SOF_INTEL_IPC4...
Checking sound/soc/sof/ipc.c: CONFIG_SND_SOC_SOF_IPC3...
```